conf/messages

```diff
@@ -499,7 +498,7 @@ tai.income.calculation.updateLink.regular=update your estimated taxable income
 tai.income.details.addMissingCompanyCar=Add a missing company car
 tai.income.details.benefit.amount.announce=- The amount of this benefit is {0}.
 tai.income.details.benefit.name.announce=Benefit type:
-tai.income.details.checkPaymentsReceived.screenReader=Check payments received from {0}.
+tai.income.details.checkPaymentsReceived.with.employer=Check payments received from {0}
```

Comment:
Is removing the full stop intentional? As you have kept it in `tai.income.details.yourEstimatedTaxableIncome.with.amount`

Reply:
yeah as its a link

Reply:
![image](https://user-images.githubusercontent.com/17907161/101472273-4f88fd00-3940-11eb-906c-4b559e53efb6.png)